packages/wp-now/src/wp-now.ts

```diff
@@ -44,34 +46,32 @@ async function applyToInstances(phpInstances: NodePHP[], callback: Function) {
 
 export default async function startWPNow(
 	options: Partial<WPNowOptions> = {}
-): Promise<{ php: NodePHP; phpInstances: NodePHP[]; options: WPNowOptions }> {
+): Promise<{
+	php: NodePHP;
+	phpInstances: NodePHP[];
+	options: WPNowOptions;
+	pool: NodePool;
+}> {
 	const { documentRoot } = options;
+
 	const nodePHPOptions: PHPLoaderOptions = {
 		requestHandler: {
 			documentRoot,
 			absoluteUrl: options.absoluteUrl,
-			isStaticFilePath: (path) => {
```

Comment:
Removing this triggers a lint error above – `seemsLikeAPHPFile` is now dead code.

Reply:
It is expected to be removed, you'll just need to remove this now-unused function, too:

<img width="714" alt="CleanShot 2023-12-22 at 13 18 27@2x" src="https://github.com/WordPress/playground-tools/assets/205419/8509e07c-2558-48ce-8c39-ca2da6583554">

Reply:
@adamziel That's been removed.